numpy/core/tests/test_deprecations.py

```diff
@@ -618,3 +618,23 @@ def test_not_deprecated(self):
             self.assert_not_deprecated(round, args=(scalar,))
             self.assert_not_deprecated(round, args=(scalar, 0))
             self.assert_not_deprecated(round, args=(scalar,), kwargs={'ndigits': 0})
+
+
+class FlatteningConcatenateUnsafeCast(_DeprecationTestCase):
+    message = "concatenate with `axis=None` will use same-kind casting"
+
+    def test_deprecated(self):
+        self.assert_deprecated(np.concatenate,
+                args=(([0.], [1.]),),
+                kwargs={'axis': None, 'out': np.empty(2, dtype=np.int64)})
```

Comment:
nit: looks more like kwargs as
```suggestion
                kwargs=dict(axis=None, out=np.empty(2, dtype=np.int64)))
```